tests/console/validate_vnc_from_target.pm

```diff
@@ -0,0 +1,31 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: Validate that, after remote installation, the xvnc.socket
+# service is active and the firewall allows tigervnc service.
+#
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+use base "consoletest";
+use strict;
+use warnings;
+
+use testapi;
+use utils "systemctl";
+
+sub run {
+    select_console "root-console";
+    # Check if the xvnc.socket is enabled and active.
+    record_soft_failure "bsc#1177485" unless (systemctl "is-active xvnc.socket", ignore_failure => 1);
```

Comment:
s/unless/if/  ?

Reply:
I believe "unless" is the correct condition to use. I assume "systemctl..." with return true is successful. I'm waiting for the results to verify.

Reply:
hmmm... looking at other modules now makes me think that I'm wrong. I will correct it after the VR finishes

Reply:
I didn't test it, maybe I'm wrong, the systemctl return `script_run` which will be 1 or 0, 0 when success and 1 when fails, so you want to record the soft failure when 1, so `if (1)` but now is `unless (1)` which is the opposite. make sense?

Reply:
trick here is always compare with 0 or 1 explicitly in the code even if make it more ugly, but for ret code helps with the boolean logic.

Reply:
I run a small test, It will return 3 if the service is inactive and null if active. I changed the code so that the soft failure will happen only for opensuse. I hope that's ok as this is a temporary workaround.

Reply:
We should remove the condition `is_opensuse`, just as a good habit to not use them in our tests. If fails for SLE could be related. Anyway this would be fix really soon.

Reply:
Done